app/javascript/widget/components/ChatFooter.vue

```diff
@@ -1,36 +1,83 @@
 <template>
-  <footer class="footer">
+  <footer v-if="!hideReplyBox" class="footer">
     <ChatInputWrap
       :on-send-message="handleSendMessage"
       :on-send-attachment="handleSendAttachment"
     />
   </footer>
+  <custom-button
```

Comment:
`woot-button`?

Reply:
We are not using ```woot-button``` on the widget side anywhere. Will raise a separate PR for this